### ISSUE TITLE
VELL-E (model_train_stage 2) error output "No such file pytorch_model.bin"

### ISSUE THREAD

**Issue by xinkez** [open-mmlab/Amphion#67]
Hi，

When I train VALL-E. I successfully trained the AR model，however when I tried to train the NAR model by "sh egs/tts/VALLE/run.sh --stage 2 --model_train_stage 2 --ar_model_ckpt_dir [ARModelPath] --name [YourExptName]
", it outputted "No such file pytorch_model.bin" in ARModelPath. In the ARModelPath, I find that only "ckpts.json model.safetensors optimizer.bin random_state_0.pkl scheduler.bin" exist. There is no "pytorch_model.bin".
I try to debug the codes, cannot find the codes used to save "pytorch_model.bin".

Pls help check it. Thanks in advance.

**Comment by lixuyuan102** [comment]
accelerate version >= 0.25 change the .bin to .safetensors. 

just replace "pytorch_model.bin" with "model.safetensors"

**Comment by lixuyuan102** [comment]
My AR model still can't accurately predict speech duration after 20 epochs/ 800K steps of training.  Did you get reasonable results with your AR model?

**Comment by xinkez** [comment]
> accelerate version >= 0.25 change the .bin to .safetensors.
> 
> just replace "pytorch_model.bin" with "model.safetensors"


I replaced "pytorch_model.bin" with "model.safetensors", however it output such errors below. Do this method work for you?

![image](https://github.com/open-mmlab/Amphion/assets/13937138/0cccbd7a-bd4b-46c6-808b-5030ea65cd1f)

**Comment by xinkez** [comment]
> My AR model still can't accurately predict speech duration after 20 epochs/ 800K steps of training. Did you get reasonable results with your AR model?

I noticed that the train loss decreased normally. how did you check the predicted speech duration? I can check it.

**Comment by lixuyuan102** [comment]
> > accelerate version >= 0.25 change the .bin to .safetensors.
> > just replace "pytorch_model.bin" with "model.safetensors"
> 
> I replaced "pytorch_model.bin" with "model.safetensors", however it output such errors below. Do this method work for you?
> 
> ![image](https://private-user-images.githubusercontent.com/13937138/292944062-0cccbd7a-bd4b-46c6-808b-5030ea65cd1f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM2NDUyOTUsIm5iZiI6MTcwMzY0NDk5NSwicGF0aCI6Ii8xMzkzNzEzOC8yOTI5NDQwNjItMGNjY2JkN2EtYmQ0Yi00NmM2LTgwOGItNTAzMGVhNjVjZDFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjI3VDAyNDMxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZmNThkYzVmZmNhY2RlOTUzMWE4YTA5NjRhOGEwNWIzYTFlY2ExNmU0N2JiMmJhNTE3MzBiNjU0NDI3YWEwMTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dsFVbqZmbtA5a4D0pzqFn6rkWxnCeXOPzYo97vb1G8g)

Sorry, my mistake. Try like this: 
![1703644902663](https://github.com/open-mmlab/Amphion/assets/137909130/e18f0f56-ef4c-4530-bcb6-9511e174e1c1)

**Comment by lixuyuan102** [comment]
> > My AR model still can't accurately predict speech duration after 20 epochs/ 800K steps of training. Did you get reasonable results with your AR model?
> 
> I noticed that the train loss decreased normally. how did you check the predicted speech duration? I can check it.

I just give the original speech and repeat the text once. And then observe if the generated speech duration matches the original speech.

**Comment by HarryHe11** [comment]
The error appears to be related to a change in the file extension in newer versions of the 'accelerate' package. To resolve this, you can revert to a previous version by running `pip install accelerate==0.24.1`. This version should be compatible with your current setup.

**Comment by RMSnow** [comment]
Hi all, it looks like the bug has been fixed. If you have any further questions, feel free to re-open this issue, and we're glad to follow up!
